data/json/items/ranged.json

```diff
@@ -3814,28 +3815,30 @@
     "symbol": "(",
     "color": "dark_gray",
     "name": "RM51 assault rifle",
-    "description": "Barely making it past the prototype stage before the world ended, the Rivtech RM51 assault rifle was designed for durability and ease of use under less than ideal circumstances.",
+    "description": "Originally produced for military use, the Rivtech RM51 assault rifle was designed for durability and ease of use under less than ideal circumstances, with its considerable bulk minimized by a compact and ergonomic layout.  Accepts box magazines.",
     "price": 1800000,
     "material": ["superalloy", "ceramic"],
     "flags": ["MODE_BURST", "WATERPROOF_GUN", "NEVER_JAMS"],
     "skill": "rifle",
     "ammo": "8x40mm",
-    "weight": 3400,
-    "volume": 9,
+    "weight": 2850,
+    "volume": 7,
     "bashing": 12,
     "cutting": 0,
     "to_hit": -1,
     "ranged_damage": 5,
     "dispersion": 90,
     "sight_dispersion": 60,
-    "aim_speed" : 6,
-    "recoil": 0,
+    "aim_speed" : 5,
+    "recoil": 75,
     "durability": 9,
     "burst": 5,
-    "clip_size": 50,
-    "reload": 500,
+    "reload": 100,
     "loudness" : 150,
-    "valid_mod_locations": [[ "accessories", 4 ],[ "barrel", 1 ],[ "conversion", 1 ],[ "grip", 1 ],[ "magazine", 2 ],[ "mechanism", 4 ],[ "muzzle", 1 ],[ "rail", 1 ],[ "sights", 1 ],[ "stock", 1 ],[ "underbarrel", 1 ]]
+    "magazines" : ["8x40_50_mag", "8x40_100_mag"],
+    "magazine_well" : 2,
+    "default_mods": [ "8x40_50_mag" ],
```

Comment:
Do we have any sort of alternative at the moment, or would they be spawning in without magazines?